python/tests/test_parquet.py

```diff
@@ -13,101 +13,132 @@
 # limitations under the License.
 
 import glob
+import math
 
-import cudf
-import cupy
-import dask_cudf
 import legate.core
+import numpy as np
+import pyarrow as pa
+import pyarrow.parquet as pq
 import pytest
 from legate.core import get_legate_runtime
 
 from legate_dataframe import LogicalColumn, LogicalTable
 from legate_dataframe.lib.parquet import parquet_read, parquet_read_array, parquet_write
 from legate_dataframe.lib.replace import replace_nulls
-from legate_dataframe.testing import assert_frame_equal, std_dataframe_set
-
-
-@pytest.mark.parametrize("df", std_dataframe_set())
+from legate_dataframe.testing import (
+    assert_arrow_table_equal,
+    assert_frame_equal,
+    std_dataframe_set_cpu,
+)
+
+
+def write_partitioned_parquet(table, path, npartitions=1):
+    if table.num_rows == 0:
+        pq.write_table(table, f"{path}/part-0.parquet")
+    partition_size = int(math.floor(table.num_rows / npartitions))
```

Comment:
I guess you initially might had a `ceil` here, just use:
```suggestion
    partition_size = table.num_rows // npartitions
```

(Feels like there should be a trick to do this shorter, but it's not worth searching :))